backend/routes.test.js

```diff
@@ -8,7 +8,7 @@ const { oauthApiFactory } = require('./api/oauthApi')
 
 const errorResponse = {
   response: {
-    data: {
+    body: {
```

Comment:
with the old version of jest this file was never being run in the build.  It would never have worked anyway as `data` had been moved to `body` due to superagent changes and some of the routes are not there any more too